### ISSUE TITLE
TypeError: connection.query(...).on is not a function

### ISSUE THREAD

**Issue by cybermerlin** [sequelize/sequelize-auto#194]
```
sequelize-auto -h 127.0.0.1 -d log-dev -u pguserhipster -x 123 -C -p 5432 -o out -e postgres
```

**Comment by geekflyer** [comment]
@cybermerlin encountered the same problem. The root cause is that sequelize doesn't yet support pg@7.x (unfortunately that's nowhere documented). You just have to downgrade to pg@6.4.2 .

**Comment by geekflyer** [comment]
actually when using `6.4.2` I'm getting a deprecationwarning which hints on the upcoming problem in in 7.x:
```
DeprecationWarning: Using the automatically created return value from client.query as an event emitter is deprecated and will be removed in pg@7.0. Please see the upgrade guide at https://node-postgres.com/guides/upgrading
```

**Comment by cybermerlin** [comment]
unfortunately, I use pg.7 with pg-bricks in other classes (old code)

**Comment by stonedraider** [comment]
downgrading to pg@6.4.2 doesn't help for me

**Comment by bpb27** [comment]
downgrading worked from me - `npm install -g pg@6.4.2`, using node 8.1.2 and npm 5.5.1

**Comment by jfabiani** [comment]
npm install -g pg@6.4.2 does not work for me!  This was very disappointing!

**Comment by Toub** [comment]
Downgrading works for me using node v8.9.4 and npm 5.6.0.

**Comment by projetoCopa** [comment]
Thanks, npm install -g pg@6.4.2 works for me!

**Comment by steveschmitt** [comment]
Fixed in `sequelize-auto@0.5.0`
